index.html

```diff
@@ -6,51 +6,71 @@
 
-        <main>
-    <section class="section-headline">
-    <div class="main-text Developer">
-    <div class="introduction">
-    <h1>Hey There. <br> I’m Ayodele</h1>
-    <h3>I’am a Software Developer</h3>
-    <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
-    
+<!-- The content part of the code begins here -->
+<main class="flexcontainer-content">
+    <section class="box-content headline">
+    <h1>Hey There. <br> I'm Ayodele</h1> 
+    <div class="box-content product">
+        <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hestiate to contact me.</p>
     </div>
+    <!-- <p class="box-content skill">I'm a software Developer!</p>
+        <P class="box-content product">I can help you build a product, feature or <br> website. Look through some of
+        my work <br> and Experience! if you like what you see <br> and have a project you need coded, <br> dont
+        hesitate to contact me.
+    </p> -->
+    <!-- Code for social media -->
+    <div class="box-content social-media"><img src="images/img_social.png" alt="icons"></div>
+    <div id="background"></div>
+    </section>
+    <!-- The second section of the code begins here  -->
+    <section class="gym-fit">
+        <div class="works">
+            <h4>My Recent Works</h4>
+        </div>
 
-    <ul class=" social-media social-media-icons mv-social-media mv-social-media-icons">
-        <li class="social-1"><a href="#"><i class="fab fa-github fa-lg"></i></a></li>
-        <li class="social-2"><a href="#"><i class="fab fa-linkedin-in fa-lg"></i></a></li>
-        <li class="social-3"><a href="#"><i class="fab fa-angellist fa-lg"></i></a></li>
-        <li class="social-4"><a href="#"><i class="fab fa-twitter fa-lg"></i></a></li>
-        <li class="social-5"><a href="#"><i class="fab fa-medium-m"></i></a></li>
-        </ul>
+        <hr class="solid"/>
 
+    <div class="pictures"></div>
+    <p id="post">Multiple-Post Stories</p>
+    <p class="post-text">
+        A daily selection of privately Personalized Reads; no account or signup
+        required has been the industry standard dummy text ever since 1500s,
+        when an unknown printer took a dummy text.
+    </p>
+    <!-- Use grid for this in a list  -->
+    <div class="grid">
+    <ul class="grid-container">
+        <li class="grid-css">css</li>
+        <li class="grid-item">html</li>
+        <li class="grid-item">bootstrap</li>
+        <li class="grid-item">Ruby</li>
+    </ul>
     </div>
-    
-    <div class="hero-image">
-        <img src="images/background-mobile.svg" alt="" />
-    </div>
+    <!-- Add a submit button -->
+    <button id="button" type="submit">See Project</button>
     </section>
+    <br>
 </main>
```

Comment:
- [ ] You've done great work to implement the Projects section, however, as part of the requirements, we will need you to implement the About myself section as well. Right now you don't have any part of the About myself section. **Please** check the requirements and ensure that you implement the About myself section of this project as well. 

## 👇About section from Figma template:
![image](https://user-images.githubusercontent.com/18730106/143686906-7b6c3e73-4aea-4260-b727-4052b547f1c1.png)

### **It is crucial that you implement this section of the page**